### PR TITLE
Update monokai because new true color breaks it

### PR DESCRIPTION
**Pull request by securisec**

### CODE REVIEW

**Comment by XVilka** [comment]
Just use "e scr.truecolor=2" in your config file

**Comment by radare** [comment]
the new rcons rewrite broke the bgcolor support. we shuold fix that instead of fixing the themes. cc @xarkes

**Comment by securisec** [comment]
@XVilka its not about setting the config value. The new rcons is breaking a lot of the color pallets regardless if truecolor = 1 or 2 by making them too dark. it also lost the ability to use two colors (one for bg and fg).

**Comment by xarkes** [comment]
As of master, using both bg and fg colors is broken. I will add a patch before 2.4.0 to fix this.

**Comment by securisec** [comment]
@xarkes you can check out the monokai theme which is the only true color one. Else, just cycle through the color themes and you will see that there are about 4 or 5 of the themes are are not showing the colors correctly, or cant see anything (black terminal) when e scr.color = 2 or 3. @XVilka it should be ok to merge the corrections to the monokai theme because i made some additional adjustments to add the monokai yellow

**Comment by xarkes** [comment]
With this PR https://github.com/radare/radare2/pull/9498 everything should be as before.

**Comment by radare** [comment]
should be fixed now, pls confirm

**Comment by securisec** [comment]
Testing this with latest pull and monokai e scr.color=2. Doesnt appear fully fixed. 

image

**Comment by xarkes** [comment]
Parsing issue, let me add a quick fix.

**Comment by xarkes** [comment]
I fixed HTML color code parsing though, so adding just a '#' is now working.
2018-02-25-093800_734x399_scrot

**Comment by xarkes** [comment]
Which looks better than the rgb:777 you put there imho :p

**Comment by radare** [comment]
Why is this not parsed as a comment?

**Comment by xarkes** [comment]
Because the command parser is not a correct parser and never parses '#' after ec as comments? :D

**Comment by radare** [comment]
The syntax should be rgb:rrggbb for those cases not with a # otherwise will break soon or late again

File: libr/cons/d/monokai [hunk 1]
```diff
@@ -1,15 +1,15 @@
-ec b0x00 rgb:222
+ec b0x00 rgb:555
 ec b0x7f rgb:fff
 ec b0xff rgb:eee
 ec other rgb:888
 ec bin rgb:888
-ec btext ef8d1a
-ec call rgb:000000 rgb:ef8d1a
+ec btext rgb:777
```
Comment (xarkes) [inline libr/cons/d/monokai]: rgb:ef8d1a

Comment (xarkes) [reply libr/cons/d/monokai]: Just update this because 777 is too ugly imho (not very different from bin or other) if you agree.
After that I can merge.

**Pull request closed without merging**